app/src/main/res/values-sw360dp/dimens.xml

```diff
@@ -6,4 +6,6 @@
     <dimen name="top_site_textview_width">65dp</dimen>
 
     <dimen name="menu_layout_margin">20dp</dimen>
+    <dimen name="firstrun_icon_bottom_margin">40dp</dimen>
```

Comment:
I think height is slightly more appropriate for this specific case here. Maybe h600?